client/src/pages/ProjectCreate.tsx

```diff
@@ -121,6 +121,7 @@ const ProjectCreate = (): JSX.Element => {
             events: [],
             tasks: [],
             creationTime: new Date(),
+            isPublic: false,
```

Comment:
currently, `ProjectCreate` in project.go controller overrides ispublic to always be true, might need to change that or this